### PR TITLE
inline iiif logo svg and style for primary and secondary ui options

### PR DESCRIPTION
**Pull request by barmintor**
- DLC-957

### CODE REVIEW

File: app/javascript/stylesheets/_base_layout.scss [hunk 1]
```diff
@@ -1294,9 +1294,30 @@ h2[itemprop="name"] {
...
-	margin-top:-3px;
+	svg {
+		height: 1em;
+		width: auto;
+		margin-top:-3px;
+	}
+}
+.btn-primary {
+	svg {
+		background-color: map-get($theme-colors, 'primary');;
```
Comment (er-k) [inline app/javascript/stylesheets/_base_layout.scss:1305]: double ;;

File: app/javascript/stylesheets/_base_layout.scss [hunk 2]
```diff
@@ -1294,9 +1294,30 @@ h2[itemprop="name"] {
...
+		path {
+			fill: map-get($theme-text-colors, 'primary');
+			stroke: map-get($theme-border-colors, 'primary');
+		}
+		flood-color: map-get($theme-colors, 'primary')
+	}
+}
+.btn-secondary {
+	svg {
+		background-color: map-get($theme-colors, 'secondary');;
```
Comment (er-k) [inline app/javascript/stylesheets/_base_layout.scss:1315]: ;;

**Review by er-k — APPROVED** [review]

File: app/javascript/stylesheets/_base_layout.scss (hunk 1 above)
Comment (barmintor) [inline app/javascript/stylesheets/_base_layout.scss:1305]:
```suggestion
		background-color: map-get($theme-colors, 'primary');
```

File: app/javascript/stylesheets/_base_layout.scss (hunk 2 above)
Comment (barmintor) [inline app/javascript/stylesheets/_base_layout.scss:1315]:
```suggestion
		background-color: map-get($theme-colors, 'secondary');
```

**Pull request merged**